tls/s2n_signature_scheme.c

```diff
@@ -379,26 +379,26 @@ const struct s2n_signature_preferences s2n_signature_preferences_20210816 = {
 };
 
 const struct s2n_signature_scheme* const s2n_sig_scheme_pref_list_rfc9151[] = {
-        /* ECDSA - TLS 1.3 */
-        &s2n_ecdsa_secp384r1_sha384,
+    /* ECDSA - TLS 1.3 */
+    &s2n_ecdsa_secp384r1_sha384,
 
-        /* RSA PSS - TLS 1.3 */
-        &s2n_rsa_pss_pss_sha384,
+    /* RSA PSS - TLS 1.3 */
+    &s2n_rsa_pss_pss_sha384,
 
-        /* ECDSA - TLS 1.2 */
-        &s2n_ecdsa_sha384, /* same iana value as TLS 1.3 s2n_ecdsa_secp384r1_sha384 */
+    /* ECDSA - TLS 1.2 */
+    &s2n_ecdsa_sha384, /* same iana value as TLS 1.3 s2n_ecdsa_secp384r1_sha384 */
 
-        /* RSA */
-        &s2n_rsa_pss_rsae_sha384,
+    /* RSA */
+    &s2n_rsa_pss_rsae_sha384,
 
-        &s2n_rsa_pkcs1_sha384,
+    &s2n_rsa_pkcs1_sha384,
 };
 
 const struct s2n_signature_scheme* const s2n_cert_sig_scheme_pref_list_rfc9151[] = {
-        /* ECDSA - TLS 1.3 */
-        &s2n_ecdsa_secp384r1_sha384,
+    /* ECDSA - TLS 1.3 */
+    &s2n_ecdsa_secp384r1_sha384,
 
-        /* RSA PSS
+    /* RSA PSS
          * https://github.com/aws/s2n-tls/issues/3435
          *
          * The Openssl function used to parse signatures off certificates does not differentiate
```

Comment:
Messed up comment. Can you grep for these automatically somehow?

Reply:
Maybe?